src/subgraph/Sales/index.ts

```diff
@@ -38,7 +38,7 @@ export const salesQuery = `
       status
       startDate
       endDate
-      tokenAmount
+      tokensForSale
```

Comment:
This is `sellAmount`

Reply:
I'm looking at the aqua-xdai-dev subgraph and there is no parameter available for sellAmount in fairsale?